pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/NotOperatorTransformFunctionTest.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.operator.transform.function;
+
+import org.apache.pinot.common.request.context.ExpressionContext;
+import org.apache.pinot.common.request.context.RequestContextUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class NotOperatorTransformFunctionTest extends BaseTransformFunctionTest {
+  // Test not true returns false.
+  @Test
+  public void testNotTrueOperatorTransformFunction() {
+    ExpressionContext expr =
+        RequestContextUtils.getExpression(String.format("Not (%d = %d)", _intSVValues[0], _intSVValues[0]));
+    TransformFunction func = TransformFunctionFactory.get(expr, _dataSourceMap);
+    Assert.assertEquals(func.getName(), "not");
+    int[] notTrueExpectedIntValues = new int[NUM_ROWS];
+    for (int i = 0; i < NUM_ROWS; i++) {
+      notTrueExpectedIntValues[i] = 0;
+    }
+    testTransformFunction(func, notTrueExpectedIntValues);
+  }
+
+  // Test not false returns true.
+  @Test
+  public void testNotFalseOperatorTransformFunction() {
+    ExpressionContext expr =
+        RequestContextUtils.getExpression(String.format("Not (%d != %d)", _intSVValues[0], _intSVValues[0]));
+    TransformFunction func = TransformFunctionFactory.get(expr, _dataSourceMap);
+    Assert.assertEquals(func.getName(), "not");
+    int[] notTrueExpectedIntValues = new int[NUM_ROWS];
+    for (int i = 0; i < NUM_ROWS; i++) {
+      notTrueExpectedIntValues[i] = 1;
+    }
+    testTransformFunction(func, notTrueExpectedIntValues);
+  }
```

Comment:
can you also add a test for values?

e.g. 
```
        RequestContextUtils.getExpression(String.format("Not (%s != %d)", INT_SV_COLUMN, _intSVValues[0]));
```
and see if it works (as they should because the expression result in a binary result

Reply:
Done